services/statsService.js

```diff
@@ -57,38 +57,44 @@ ORDER BY
   try {
     const customStats = await getCustomStats();
 
-    const overriddenData = data.map(d => {
-      const customCountryStat = customStats.find(c => c.countryCode && d.countryCode && c.countryCode.toLowerCase() === d.countryCode.toLowerCase());
+    const overriddenData = [];
+    if(date) {
+      overriddenData.push(...data);
+    }
+    else {
+      overriddenData.push(...data.map(d => {
+        const customCountryStat = customStats.find(c => c.countryCode && d.countryCode && c.countryCode.toLowerCase() === d.countryCode.toLowerCase());
```

Comment:
you can write in this way
```
let overriddenData;
if (date) {
  overriddenData = data;
}
else {
  overriddenData = data.map(d => {
  ...
}
```

Reply:
@xming13 Yea, I feel that was bad code, thanks on corretion. But except that there is another question about this PR:
I see that /v3/analytics/country has been removed from API, but there is /v3/stats/worldometer/country method.

Can I add optional date param to last one mathod?

Reply:
@clearsense yeah sure